ci/Dockerfile

```diff
@@ -7,4 +7,4 @@ RUN apt-get install wget
 RUN wget -q -O - https://raw.githubusercontent.com/starkandwayne/homebrew-cf/master/public.key | apt-key add -
 RUN echo "deb http://apt.starkandwayne.com stable main" | tee /etc/apt/sources.list.d/starkandwayne.list
 RUN apt-get update
-RUN apt-get install jq spruce git
+RUN apt-get -y install jq spruce git vim
```

Comment:
Is this a leftover from debugging?

Reply:
yes but I thought we might as well leave it to make debugging easier in the future

Reply:
ok then. I think it won't add to much to the image.